includes/embeds/class-amp-youtube-embed.php

```diff
@@ -55,23 +83,49 @@ public function shortcode( $attr ) {
 
 		$video_id = $this->get_video_id_from_url( $url );
 
-		return $this->render( array(
-			'url' => $url,
-			'video_id' => $video_id,
-		) );
+		return $this->render(
+			array(
+				'url'      => $url,
+				'video_id' => $video_id,
+			)
```

Comment:
Aside: things like this are more elegant as `compact( 'url', 'video_id' )`

Reply:
Agreed that compact is better. `phpcbf` reformatted it as above (expanded).